### PR TITLE
Spellchecks comments and members, removes unused using statements and…

### PR DESCRIPTION
**Pull request by MihaMarkic**
**Avalonia.Controls project**

Spell checks:
* comments
* members (some private methods and private and protected methods' arguments affected)

Also
* removes unused using statements
* sorts using statements

### CODE REVIEW

File: src/Avalonia.Controls/Platform/ITopLevelImpl.cs [hunk 1]
```diff
@@ -20,12 +20,12 @@ namespace Avalonia.Platform
     public interface ITopLevelImpl : IDisposable
     {
         /// <summary>
-        /// Gets the client size of the toplevel.
```
Comment (jmacato) [inline src/Avalonia.Controls/Platform/ITopLevelImpl.cs]: I think toplevel is a proper term for this

Comment (MihaMarkic) [reply src/Avalonia.Controls/Platform/ITopLevelImpl.cs]: I was in doubt about that one. Anyway, reverted.

**Comment by siegfriedpammer** [comment]
One thing I noticed:

https://github.com/AvaloniaUI/Avalonia/blob/ad69128b4efab0ef8d9c4eb806d484fdd483f713/src/Avalonia.Base/ValueStore.cs#L171-L180

`_defferedSetter` => `_deferredSetter`. Maybe you could include that as well? 😃

**Comment by MihaMarkic** [comment]
@jmacato Eventually.

**Comment by MihaMarkic** [comment]
Guys, it this a go for merging? Should I proceed with other assemblies?

**Review by jmacato — APPROVED** [review]

**Comment by jmacato** [comment]
@MihaMarkic if you're going to do more corrections on other assemblies then i'll leave this open till you're done

**Comment by MihaMarkic** [comment]
@jmacato Perhaps it's better going in batches rather than everything in a single commit?

**Comment by jmacato** [comment]
@MihaMarkic I wholeheartedly agree, will merge this now :)

**Pull request merged**